interfaces/cython/cantera/kinetics.pxd

```diff
@@ -69,6 +69,7 @@ cdef extern from "cantera/kinetics/InterfaceKinetics.h":
     cdef cppclass CxxInterfaceKinetics "Cantera::InterfaceKinetics":
         void advanceCoverages(double, double, double, double, size_t, size_t) except +translate_exception
         void solvePseudoSteadyStateProblem() except +translate_exception
+        double InterfaceCurrent(int) except +translate_exception
```

Comment:
If the declaration in `InterfaceKinetics.h` is changed, then change accordingly, here.